languages/ocaml/menhir/parse_ml.ml

```diff
@@ -40,7 +40,7 @@ let tokens file =
 (*****************************************************************************)
 let parse filename =
   let stat = Parsing_stat.default_stat filename in
-  let toks = tokens filename in
+  let toks = tokens (Parsing_helpers.File filename) in
```

Comment:
Please rename the argument to the tokens function above too from 'file' to 'target' (or 'target_kind')